integration-tests/integration/features/outcomes.feature

```diff
@@ -12,4 +12,12 @@ Feature: Outcomes List
     And I should see the Primary navigation
     And I should see the Primary navigation "Outcomes" link
     When I click on the "Outcomes" link in the Primary navigation
-    And the page should produce a 404 error
\ No newline at end of file
+    Then I should be on the "Hearing outcomes" page
+
+    And I should see the phase banner
+    And I should see the tag "Feedback"
+    And I should see phase banner link "Give us your feedback" with href "https://docs.google.com/forms/d/e/1FAIpQLScluoDOXsJ_XBO3iOp283JE9mN3vTVNgEJcPNDHQQvU-dbHuA/viewform?usp=sf_link"
+    
+    And I should see a tab with text "Cases to result"
```

Comment:
Also add - on clicking the tab should land on corresponding location?

Reply:
Have added that now